crates/suiop-cli/src/cli/incidents/mod.rs

```diff
@@ -17,7 +17,7 @@ use jira::generate_follow_up_tasks;
 use pd::print_recent_incidents;
 use selection::review_recent_incidents;
 use std::path::PathBuf;
-use tracing::debug;
+use tracing::{debug, info};
```

Comment:
why both here?

Reply:
These are just macros to log at different levels. You can see the calls to `info!()` and `debug!()` in the code below if you expand it